LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016 Alexandru Somai and Jenkins contributors
```

Comment:
Makes sense to do it since a significant amount of code has been taken from Copy Artifact plugin